geth/jail/jail_test.go

```diff
@@ -115,6 +128,46 @@ func (s *JailTestSuite) TestFunctionCall() {
 	require.Equal(expectedResponse, response)
 }
 
+func (s *JailTestSuite) TestJailRPCAsyncSend() {
+	require := s.Require()
+
+	defer func() {
+		if err := recover(); err != nil {
+			require.NotNil(err, "Panic occured")
+		}
+	}()
+
+	require.NotNil(s.jail)
+
+	s.StartTestNode(params.RopstenNetworkID)
+	defer s.StopTestNode()
+
+	// load Status JS and add test command to it
+	s.jail.BaseJS(baseStatusJSCode)
+	s.jail.Parse(testChatID, rxSendJS)
+
+	cell, err := s.jail.GetCell(testChatID)
+	require.NoError(err)
+	require.NotNil(cell)
+
+	var wg sync.WaitGroup
+
+	// internally (since we replaced `web3.send` with `jail.Send`)
+	// all requests to web3 are forwarded to `jail.Send`
+	for i := 0; i < 5; i++ {
+		go func() {
+			wg.Add(1)
+			defer wg.Done()
+
+			_, err = cell.Run(txSendAsync)
+
+			require.NoError(err, "Request failed to process")
+		}()
+	}
+
+	wg.Wait()
+}
```

Comment:
Please, remove extra (almost all) new lines in the test. I see that you're striving to better indenting and readability but in this case you should group lines executing the same function together and add new lines when groups of code serve different functions.
Generally, there's nothing bad in writing code every line, it doesn't become less readable in case it's grouped by meaningful blocks.